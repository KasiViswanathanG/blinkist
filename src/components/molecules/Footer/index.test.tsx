import {screen, fireEvent, getByText, render } from '@testing-library/react'
import React from 'react'
import renderer from 'react-test-renderer'

import FooterComponent from '.'


describe('Testing the FooterComponent', () => {
    it('render FooterComponent', () => {
      const footer = renderer.create(<FooterComponent />).toJSON();
      expect(footer).toMatchSnapshot();
    });
})  