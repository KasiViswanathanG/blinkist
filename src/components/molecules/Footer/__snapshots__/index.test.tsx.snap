// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the FooterComponent render FooterComponent 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-102mbgj-MuiPaper-root"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <div
      style={
        Object {
          "display": "flex",
          "justifyContent": "space-around",
        }
      }
    >
      <div
        style={
          Object {
            "marginTop": "40px",
          }
        }
      >
        <img
          alt="logo"
          className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img css-jqh5pz-MuiCardMedia-root"
          src="static/images/logo.png"
        />
        <p
          className="MuiTypography-root MuiTypography-body1 css-eaf4qi-MuiTypography-root"
        >
          Big ideas in small packages
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 css-wmy4kh-MuiTypography-root"
        >
          Start learnign now
        </p>
      </div>
      <div
        style={
          Object {
            "marginTop": "40px",
          }
        }
      >
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-um3j9i-MuiTypography-root"
        >
          Editorial
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Book lists
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          What is Nonfiction?
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          What to read next?
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Benefits of reading
        </p>
      </div>
      <div
        style={
          Object {
            "marginTop": "40px",
          }
        }
      >
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-um3j9i-MuiTypography-root"
        >
          Useful links
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Pricing
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Blinkist business
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Gift cards
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Blinkist magaine
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Contact & help
        </p>
      </div>
      <div
        style={
          Object {
            "marginTop": "40px",
          }
        }
      >
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-um3j9i-MuiTypography-root"
        >
          Company
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          About
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Careers
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          partners
        </p>
        <p
          className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-1vnvo1s-MuiTypography-root"
        >
          Code of Conduct
        </p>
      </div>
    </div>
    <div
      style={
        Object {
          "marginLeft": "65px",
          "marginTop": "40px",
        }
      }
    >
      <p
        className="MuiTypography-root MuiTypography-body1 css-1m75f5p-MuiTypography-root"
      >
        Â© Blinkist 2021 Sitemap   |   Imprint   |   Terms of Service   |   Privacy Policies
      </p>
    </div>
  </div>
</div>
`;
